dumpling/export/writer_util.go

```diff
@@ -212,20 +215,31 @@ func WriteInsert(
 
 	selectedField := meta.SelectedField()
 
-	// if has generated column
-	if selectedField != "" && selectedField != "*" {
-		insertStatementPrefix = fmt.Sprintf("INSERT INTO %s (%s) VALUES\n",
-			wrapBackTicks(escapeString(meta.TableName())), selectedField)
-	} else {
-		insertStatementPrefix = fmt.Sprintf("INSERT INTO %s VALUES\n",
-			wrapBackTicks(escapeString(meta.TableName())))
+	// Determine if we should write INSERT prefix based on chunk position
```

Comment:
I think we don't need to change old logic. As `tryToWriteTableData`, there's `chunkIdx` in input arguments. We build a new writer for a given `chunkIdx`. The INSERT INTO prefix is always written.